src/main/mpack/mpack.json

```diff
@@ -55,7 +55,7 @@
           "applicable_stacks": [
             {
               "stack_name": "HDP",
-              "stack_version": "2.6"
+              "stack_version": "3.0"
```

Comment:
This file has the property `min_ambari_version`;
do we need to change it? I think not unless one of the Solr versions get deprecated, right?

Reply:
My understanding is that our "min_ambari_version" is informed by the oldest SOLR-service version that we bundle in here. 5.5.5 is our oldest, and it's used for HDP 2.4, which in turn is intended for Ambari 2.2.1.0 (https://hortonworks.com/wp-content/uploads/2016/03/ReleaseNotes_3_3_2016.pdf)

So based on that, I'm not sure we can increase our min_ambari_version, at least not unless we want to get rid of the SOLR 5.5.5 stack version.  (In fact, I'm not sure why the min_ambari_version is as high as it is.  I'd think it would need to be 2.2.1.0.  Maybe I'm misunderstanding how these versions fit together.  Will do a little more digging.)